man/fi_verbs.7.md

```diff
@@ -151,6 +151,14 @@ requires the use of shared receive contexts. See [`fi_rxm`(7)](fi_rxm.7.html).
 To enable XRC, the following environment variables must usually be set:
 FI_VERBS_PREFER_XRC and FI_OFI_RXM_USE_SRX.
 
+### Atomics
+Verbs hardware has limited support for atomics on little-endian machines as the
```

Comment:
Does the limitation apply to Mellanox HCAs only or also RoCE NICs from other vendors?

Reply:
Sorry there's very limited documentation so that took me a while to verify... the limitation is only on Mellanox. Other vendors pass validation without conversion. Updated the patch to specify Mellanox